jobs/kola-aws.Jenkinsfile

```diff
@@ -44,23 +44,32 @@ if (s3_stream_dir == "") {
     s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
 }
 
-cosaPod(image: params.COREOS_ASSEMBLER_IMAGE,
-        memory: "256Mi", kvm: false,
-        secrets: ["aws-fcos-builds-bot-config", "aws-fcos-kola-bot-config"]) {
+try { timeout(time: 120, unit: 'MINUTES') {
```

Comment:
Maybe the commit title for this should be:

> kola-{aws,gcp,openstack}: add timeout and send slack message on failure

?

Reply:
Maybe 1h30 for AWS? Looking at past runs, it seems to take ~1h.

Reply:
done